docs/services/webex.md

```diff
@@ -0,0 +1,41 @@
+# Webex Teams
```

Comment:
Could you please add a template section?

Reply:
Webex specific templates are not implemented, you can use markdown in the message:

``` yaml
data:
  template.cert-ready: |
    message: |
      **Argo sync**
      Certificate {{.cert.metadata.name}} is ready!
```

what gives following message on webex:
<img width="239" alt="image" src="https://user-images.githubusercontent.com/33880187/205154834-02a8583e-255f-402f-b820-bf18b3000623.png">